### PR TITLE
Feature/character creation form

### PR DESCRIPTION
**Pull request by NickLiggett**
# What does this PR do?
- [x] New feature (adds functionality)
- [] Debugging/Refactor

**Implementation/Changes:**
* Configured Character Creator to implement "Add to Party" functionality and started making Monster Details Section dynamic.

**Where should your reviewer start?**
*

**What (if any) features are you implementing?**
*

**What (if anything) did you refactor?**
*

**What issues does this PR close?**
*

**Is there anything that you need from your teammate?**
*

**Any other comments, questions, or concerns?**
*

# How Has This Been Tested?
- [x] I tested my changes in the browser
- [] I tested with Cypress

# Checklist:
- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] My changes generate no new bugs
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Ant-Shell — APPROVED** [review]
Approved!

File: src/Components/CharacterCreator.tsx [hunk 1]
```diff
@@ -1,31 +1,71 @@
...
+//     about: string
+// }
+
+type Props = {
+    submitForm: (event: React.FormEvent<HTMLFormElement>, character: object) => void
+}
+
+const CharacterCreator: FC<Props> = (props: Props) => {
+
+    const [formData, setFormData] = useState<object>({name: '', race: '', classs: '', hp: 0, ac: 0, str: 0, con: 0, dex: 0, wis: 0, int: 0, cha: 0, about: ''})
```
Comment (mayakappen) [inline src/Components/CharacterCreator.tsx:26]: this makes so much more sense here thank you :)

File: src/Components/CharacterCreator.tsx [hunk 2]
```diff
@@ -1,31 +1,71 @@
...
+
+    // This can stay here ^^
+
+   const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setFormData({ ...formData, [event.target.id]: event.target.value })
```
Comment (mayakappen) [inline src/Components/CharacterCreator.tsx:31]: thanks for explaining this to me earlier!

File: src/Components/MonsterSection.tsx [hunk 3]
```diff
@@ -1,17 +1,73 @@
...
+        hit_points: 60,
+        xp: 2,
+        proficiencies: [{
+            value: 1,
+            proficiency: {
+                index: "saving-throw-dex",
+                name: "Saving Throw: DEX"
+            }
+        }],
+        special_abilities: [{name: "Amphibious" , desc: "The aboleth can breathe air and water."}]
```
Comment (mayakappen) [inline src/Components/MonsterSection.tsx:54]: so cool you're able to bring all of this in! :o

File: src/Components/Party.tsx [hunk 4]
```diff
@@ -1,15 +1,31 @@
-import React from "react"
+import React, { FC } from "react"
 import "../Styles/Party.css"
 
-const Party = () => {
+type Props = {
+    party: {name: string, race: string, classs: string, hp: number, ac: number, str: number, con: number, dex: number, wis: number, int: number, cha: number, about: string}[]
```
Comment (mayakappen) [inline src/Components/Party.tsx:5]: these make so much more sense as props here as opposed to the form i love it

File: src/Components/Party.tsx [hunk 5]
```diff
@@ -1,15 +1,31 @@
...
+
+const Party: FC<Props> = (props) => {
+
+    const partyList = props.party.map(member => {
+        return (
+            <div className="party-member">
+                <p>Name: {member.name} / Race: {member.race} / Class: {member.classs}</p>
+                <p>HP: {member.hp} / AC: {member.ac}</p>    
+                <p>STR: {member.str} / CON: {member.str} / DEX: {member.str} / WIS: {member.str} / INT: {member.str} / CHA: {member.str}</p>
+                <button>Delete</button>
```
Comment (mayakappen) [inline src/Components/Party.tsx:16]: love the layout with the / separators!

**Review by mayakappen — COMMENTED** [review]
I love everything you did! Thanks for getting that done!

**Pull request merged**